src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -148,45 +148,17 @@ private function populateAvailableType($id, Definition $definition)
             $this->types[$type] = $id;
         }
 
-        // Cannot use reflection if the class isn't set
```

Comment:
This code is a duplicate of https://github.com/hason/symfony/blob/autowire/src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php#L231-L233